paper/tsd2017.tex

```diff
@@ -40,11 +40,11 @@
 
 % TSD 2017: Put your title here (please, use capitalization, see e.g.
 % http://en.wikibooks.org/wiki/Basic_Book_Design/Capitalizing_Words_in_Titles)
-\title{Language-Independent Acoustic Modeling for Medieval Latin}
+\title{Language Specific Knowledge and Data Free Acoustic Modeling for Medieval Latin LVCSR}
```

Comment:
Nem egyszerű ez a címkérdés. A data free-t lehet úgy értelmezni, hogy nincs  összekötve  a language specific-kel?

Az világos, hogy a cikk nem csak a free approach-ról szól, de ez a jelenlegi címre ugyanígy vonatkozik, nem?

Nem ragaszkodom a mostani a címcseréhez, később is lehet, jó lenne még jobb cím. A language independent szerintem semmitmondó, már elhasználták mások és azért bele lehet kötni...